src/librustdoc/html/static/themes/main.css

```diff
@@ -383,3 +383,8 @@ kbd {
 		background: #fff;
 	}
 }
+
+.hidden-by-impl-hider, 
```

Comment:
This rule has nothing to do in themes. Please put in `rustdoc.css` at the upper level.